### ISSUE TITLE
I'm seeing these warnings while installing expo-cli .

### ISSUE THREAD

**Issue by its-shubham2323** [expo/expo-cli#3661]
`PS C:\Users\lenovo> npm i -g expo-cli
npm WARN registry Unexpected warning for https://registry.npmjs.org/: Miscellaneous Warning ETIMEDOUT: request to https://registry.npmjs.org/expo-cli failed, reason: connect ETIMEDOUT 104.16.21.35:443
npm WARN registry Using stale data from https://registry.npmjs.org/ due to a request error during revalidation.
npm WARN deprecated topo@2.0.2: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).p-ansi: sill fetch manifest
npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm WARN deprecated hoek@4.2.1: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).
npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
npm WARN deprecated uuid@3.0.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm WARN deprecated chokidar@2.1.8: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.
npm WARN deprecated chokidar@2.1.8: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.
npm WARN deprecated querystring@0.2.0: The querystring API is considered Legacy. new code should use the URLSearchParams API instead.
npm WARN deprecated uuid@2.0.3: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm WARN deprecated @hapi/pinpoint@2.0.0: Moved to 'npm install @sideway/pinpoint'
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm WARN deprecated @hapi/address@4.1.0: Moved to 'npm install @sideway/address'
npm WARN deprecated joi@11.4.0: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).
npm WARN deprecated core-js@2.6.12: core-js@<3.3 is no longer maintained and not recommended for usage due to the number of issues. Because of the V8 engine whims, feature detection in old core-js versions could cause a slowdown up to 100x even if nothing is polyfilled. Please, upgrade your dependencies to the actual version of core-js.`

I'm using Node js Version : 16.3.0
Npm version : 7.15.1

I'm just wishing to know that is it okay or it gonna trouble me while developing my app ??

**Comment by daveslutzkin** [comment]
Yeah, it's pretty messy, but you don't have to worry about these ones. Hopefully the expo team has time to chase them down soon - especially core-js.

**Comment by its-shubham2323** [comment]
> Yeah, it's pretty messy, but you don't have to worry about these ones. Hopefully the expo team has time to chase them down soon - especially core-js.

Great 🙌

**Comment by brentvatne** [comment]
We resolve npm warnings whenever we can, and usually this means upgrading or switching dependencies, but we don't always have control over this. Sometimes a package has moved to a new name and made breaking changes at the same time, for example dropping support for a specific Node version that we support in expo-cli. Usually the warnings you see are from distant downstream dependencies, eg: expo-cli depends on package `a` which depends on `x` which depends on `y` which depends on `z`. we have to wait for `a` to update their dependency on `x` which results in updating `y` and `z`.

Basically every non-trivial package in the Node ecosystem will come with messages like this on install due to the unfortunate way that unactionable warnings are exposed to users. Some tools use `ncc` to vendor the dependencies, which essentially hides the warnings but the underlying code is basically the same.

We update dependencies whenever it makes sense to, and we do that frequently. These warnings will not interfere with your usage of expo-cli :)
